devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -145,7 +145,7 @@ def parse(String description) {
 			result = zwaveEvent(cmd)
 		}
 	}
-	if (result?.name? == 'hail' && hubFirmwareLessThan("000.011.00602")) {
+	if (result?.name?.equals('hail') && hubFirmwareLessThan("000.011.00602")) {
```

Comment:
Does this work? A few of the `zwaveEvent` functions return a list.

Reply:
I believe that the `?` after result will mean it gets marked as false?

Reply:
yeah, just verified

Reply:
And `result?.name == 'hail'` is what is throwing the original exception? I would expect that to just result in `null == 'hail'` which would be false.